src/components/paradox/results/personality/career/inline-jobs/job.js

```diff
@@ -0,0 +1,67 @@
+import PropTypes from "prop-types";
+import {faLocationDot} from "@fortawesome/free-solid-svg-icons";
+import Icon from "lib/helpers/icon";
+import DangerousHTML from "lib/helpers/dangerous-html";
+import style from "./style.scss";
+
+function InlineJob({job, jobSource = "Indeed", translate}) {
+  const jobSourceURL = {
+    Indeed: "https://www.indeed.com/",
+    Monster: "https://www.monster.com",
+    MyNextMove: "https://www.mynextmove.org"
+  };
+
+  if(!job || Object.keys(job).length === 0) {
+    return (
+      <div className={`${style.inlineJob} ${style.empty}`}>
+        <div>
+          <DangerousHTML html={translate("no_jobs", {job_source: jobSource || "Indeed"})} />
```

Comment:
It shouldn't need the fallback here either

Reply:
I believe jobSource can be passed an empty string but I can fix that by adding a condition in Dash

Reply:
I think either forcing it to be present or just adding the fallback within the component would work. Like

```
function InlineJob({job, jobSource: _jobSource, translate}) {
  const jobSource = _jobSource || "Indeed";
```

That way line 23 doesn't also need a fallback

Reply:
Moved all jobSource validation to InlineJobs component! The component now returns null if an invalid input if passed in!